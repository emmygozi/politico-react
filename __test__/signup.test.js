import React from "react";
import Signup from "../src/component/signup";

const props = {
  user: {},
  isAuth: {},
  message: {},
  status: {},
  offices: {}
};

const store = mockStore({
  user: {
    user: {}
  }
});

const setup = () => {
  const wrapper = shallow(
    <Provider store={store}>
      <Signup {...props} />
    </Provider>
  );
  return wrapper;
};

describe("<Signup />", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = setup();
  });

  it("should render the Login component correctly", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
