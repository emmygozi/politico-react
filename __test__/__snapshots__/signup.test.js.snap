// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Signup /> should render the Login component correctly 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "clear": [Function],
          "get": [Function],
          "notify": [Function],
          "subscribe": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": [Function],
      },
    }
  }
>
  <Connect(Signup)
    isAuth={Object {}}
    message={Object {}}
    offices={Object {}}
    status={Object {}}
    user={Object {}}
  />
</ContextProvider>
`;
